packages/block-library/src/quote/style.scss

```diff
@@ -18,5 +18,9 @@
 			font-size: 1.125em;
 			text-align: right;
 		}
+
+	}
+	cite {
```

Comment:
Would it be worth using a direct child combinator (`> cite`) since the Quote block allows arbitrary blocks? Just in case anyone winds up using a custom block within a Quote block that expects the `cite` to be inline 🤔

Reply:
oooh right yes that's a good idea!